doc/api/http2.md

```diff
@@ -1615,6 +1615,22 @@ added: v8.4.0
 
 The `'timeout'` event is emitted when there is no activity on the Server for
 a given number of milliseconds set using `http2server.setTimeout()`.
+**Default:** 2 minutes.
+
+#### server.setTimeout([msecs][, callback])
```

Comment:
Section heading are ABC-sorted, so this and the next added headings need to be placed after the `#### server.close([callback])`.

Reply:
If an error is thrown if no callback is assigned, should we make the `callback` parameter in both signatures mandatory?